services/brawlstars.py

```
@@ -0,0 +1,450 @@
+    highest_trophies: int = 0
+    exp_level: int = 0
+    exp_points: int = 0
+    is_qualified_from_championship: bool = False
+    solo_victories: int = 0
+    duo_victories: int = 0
+    x3vs3_victories: int = 0
+    club: BrawlStarsClub | None = None
+    brawlers: list[BrawlerInfo] = Field(default_factory=list)
+
+    class Config:
+        alias_generator = lambda s: s  # accept both camelCase and snake_case
```

Flake8 (7.3.0)

[error] do not assign a lambda expression, use a def

(E731)